File: 2014-10-CSharp-Fundamentals-1/5. Conditional Statements/MultipleIfElse/MultipleIfElse.cs
using System;

class MultipleIfElse
{
    static void Main()
    {
        Console.Write("Please enter a character: ");
        char ch = (char) Console.Read();

		if (ch == 'A' || ch == 'a')
		{
			Console.WriteLine("Vowel [ei]");
		}
		else if (ch == 'E' || ch == 'e')
		{
			Console.WriteLine("Vowel [i:]");
		}
		else if (ch == 'I' || ch == 'i')
		{
			Console.WriteLine("Vowel [ai]");
		}
		else if (ch == 'O' || ch == 'o')
		{
			Console.WriteLine("Vowel [ou]");
		}
		else if (ch == 'U' || ch == 'u')
		{
			Console.WriteLine("Vowel [ju:]");
		}
		else
		{
			Console.WriteLine("Consonant");
		}

    }
}

File: 2015-10-Java-Fundamentals-1/05. Conditional-statements/demos/src/MultipleIfElse.java
import java.text.MessageFormat;
import java.util.Scanner;

public class MultipleIfElse {
    public static void main(String[] args) {
        Write("Please enter a character: ");

        Scanner scanner = new Scanner(System.in);
        char ch = scanner.next().charAt(0);

        if (ch == 'A' || ch == 'a') {
            WriteLine("Vowel [ei]");
        }
        else if (ch == 'E' || ch == 'e') {
            WriteLine("Vowel [i:]");
        }
        else if (ch == 'I' || ch == 'i') {
            WriteLine("Vowel [ai]");
        }
        else if (ch == 'O' || ch == 'o') {
            WriteLine("Vowel [ou]");
        }
        else if (ch == 'U' || ch == 'u') {
            WriteLine("Vowel [ju:]");
        }
        else {
            WriteLine("Consonant");
        }
    }

    public static void Write(Object obj) {
        System.out.print(obj);
    }

    public static void WriteLine(Object obj) {
        System.out.println(obj);
    }
}